api-reference/retrieving-an-entity.md

```diff
@@ -10,8 +10,9 @@ GET /api/v1/{entity-type}/{id} HTTP/1.1
 
 | Field           | In     | Type     | Description                      |
 | --------------- | ------ | -------- | -------------------------------- |
-| `content-type`  | header | `string` | Set to `application/json`        |
-| `authorization` | header | `string` | Set to `bearer {access-token}`   |
+| `content-type`  | header | `string` | Set to `application/json` to retrieve in json. Other allowed values include `application/vc+ld+json`, `application/ld+json`, `application/pdf`       |
+| `template-key` | header | `string` | Set appropriate value. Only required in case of `content-type` being `application/pdf`
```

Comment:
`template-key` is an optional header, it can be used for pdf/html/svg content type. It should be one of the keys mentioned in `certificateTemplates` in the scheme config.

`template` is an optional header where we can pass the URL of the external template directly in the API. To use this `enable_external_templates` ENV needs to be enabled.